ontotext-yasgui-web-component/src/components/ontotext-yasgui-web-component/ontotext-yasgui-web-component.tsx

```diff
@@ -635,6 +647,11 @@ export class OntotextYasguiWebComponent {
     this.output.emit(toOutputEvent(event));
   }
 
+  @Listen('internalCountQueryAbortedEvent')
+  onCountQueryAborted(event: CustomEvent<InternalCountQueryAbortedEvent>) {
+    this.output.emit(toOutputEvent(event));
```

Comment:
I'm not sure how useful this is, becuase the client leaves the page which in turn triggres query abort process where we can have multiple queries running. In the end here we will emit multiple query aborted events which is IMO is useless from the client point of view

Reply:
This event is also emitted when a tab is closed, in case there was an ongoing count query. This allows the client to take specific actions; in our case, in the Workbench (WB), we call an endpoint to effectively stop the running query.